Makefile

```diff
@@ -7,7 +7,7 @@ install-dev:
 	@for f in $(shell ls ${SERVICES_DIR}); do pip install -e ${SERVICES_DIR}/$${f}[dev]; done
 
 test:
-	@for f in $(shell ls ${SERVICES_DIR}); do  set -e; cd ${SERVICES_DIR}/$${f}; pytest; cd ../..; done
+	@for f in $(shell ls ${SERVICES_DIR}); do  set -e; cd ${SERVICES_DIR}/$${f}; sh -c 'pytest || ([ $$? = 5 ] && exit 0 || exit $$?)'; cd ../..; done
```

Comment:
this command is getting quite complex, isn't there a simpler way to do it?

What is this error 5 that we are skipping?

Reply:
I don't think there is a simpler way.
Error 5 means no tests have been found.

Reply:
Another option would be to remove testing entirely. We decided against auto-generating them so there probably will never be tests.

Reply:
There will be tests for the wait methods and if we want to move the core pkg to this repo there will also be those tests